packages/react-core/src/components/CalendarMonth/__tests__/CalendarMonth.test.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+import { render, screen } from '@testing-library/react';
+
+import { CalendarMonth } from '../CalendarMonth';
+
+test('Renders the first date in a month when a custom weekStart is passed', () => {
+  const formatAria = (date: Date) =>
```

Comment:
Can you add a comment here just stating that this custom aria label is needed because of a bug with automatic aria label generation as it is now, and that it can be removed once that bug is fixed?

Reply:
That's a fair point, though the same could be said for this approach and its reliance on the `cellAriaLabel` prop functioning properly. This line of thinking could also extend to many other behaviors of the component, such as the `weekStart` or `date` props working properly, or the structure of the component.

I totally get not wanting this test to be able to fail for something that isn't the actual behavior under test, but I think that it's generally reasonable for a test to rely on other behavior of the component under test since totally avoiding that would be next to impossible.

Plus if the generated aria labels are properly tested developers should be able to see that test failing as well if the generated aria label is changed, which should make it apparent what the true cause of the failure here is.

All of that being said I don't think what you've got here is bad by any means and I don't think it neccesarily needs to go, but I would still like a comment saying *why* it's here and that it shouldn't be *needed* once the bug is fixed 🙂